jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpStatus.java

```diff
@@ -205,74 +205,64 @@ public String toString()
         }
 
         /**
-         * Simple test against an code to determine if it falls into the
-         * <code>Informational</code> message category as defined in the <a
-         * href="http://tools.ietf.org/html/rfc1945">RFC 1945 - HTTP/1.0</a>,
-         * and <a href="http://tools.ietf.org/html/rfc7231">RFC 7231 -
-         * HTTP/1.1</a>.
+         * Simple test against a code to determine if it falls into the
```

Comment:
"Simple" adds nothing to this documentation:

```suggestion
         * Test to determine if the status is of the
```

Reply:
Ditto below